src/api/kube.ts

```diff
@@ -1646,11 +1660,17 @@ export class KubeHelper {
     }
   }
 
-  async deleteNamespace(namespace: string): Promise<void> {
+  async deleteNamespace(namespace: string): Promise<boolean> {
     const k8sCoreApi = KubeHelper.KUBE_CONFIG.makeApiClient(CoreV1Api)
     try {
       await k8sCoreApi.deleteNamespace(namespace)
+      return true
     } catch (e) {
+      if (e.response && e.response.body) {
+        if (e.response.body.code === 403) {
+          return false
```

Comment:
That's a bit strange. The Namespace isn't deleted in both cases but only for 403 we return false.
It seems inconsistent for me.

Reply:
True. Will fix.

Reply:
Agree, will revert boolean return change.